distribution/client/src/lib/update.js

```diff
@@ -74,7 +74,7 @@ class Update {
         updates.core.checksum.signature));
     }
 
-    if ((!updates.plugins) || (!updates.plugins.updates)) {
+    if ((!updates.plugins) || (!updates.plugins.updates && !updates.plugins.deletes)) {
       logger.debug('No plugins available in the response');
       this.updateInProgress = null;
       this.saveUpdateSync(updates);
```

Comment:
:fire_engine: 
Unrelated with your change, but isn't there a huge bug here? If only the core is updated, we'll bail out without restart nor snapshotting, but the war has been downloaded and put at the location ready isn't it?

Reply:
true, hadn't found that one yet, we have a severe lack of positive testing that I'm trying to fix